posthog/models/event/sql.py

```diff
@@ -89,6 +89,7 @@ def DROP_EVENTS_TABLE_SQL():
     , $group_4 VARCHAR MATERIALIZED {trim_quotes_expr("JSONExtractRaw(properties, '$group_4')")} COMMENT 'column_materializer::$group_4'
     , $window_id VARCHAR MATERIALIZED {trim_quotes_expr("JSONExtractRaw(properties, '$window_id')")} COMMENT 'column_materializer::$window_id'
     , $session_id VARCHAR MATERIALIZED {trim_quotes_expr("JSONExtractRaw(properties, '$session_id')")} COMMENT 'column_materializer::$session_id'
+    , $session_id_uuid Nullable(UInt128) MATERIALIZED toUInt128(accurateCastOrNull({trim_quotes_expr("JSONExtractRaw(properties, '$session_id')")}, 'UUID')) COMMENT 'column_materializer::$session_id_uuid'
```

Comment:
See this thread for debate over whether this should be nullable, or if it should be non-nullable, with invalid UUIDs set to 0.

https://posthog.slack.com/archives/C076E99B152/p1741797495838499

Reply:
```suggestion
    , $session_id_uuid Nullable(UInt128) MATERIALIZED toUInt128(JSONExtract(properties, '$session_id', 'Nullable(UUID)'))
```

(See earlier comments.)